src/components/button/_macro.njk

```diff
@@ -17,7 +17,7 @@
       {% if params.url and params.newWindow %}target="_blank" rel="noopener"{% endif %}
       {% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
       >
-      <span class="btn__inner{% if params.innerClasses %} {{ params.innerClasses }}{% endif %}{% if params.icon and params.submitType != 'loader' %} icon--{{ params.icon }}{% elif params.newWindow %} icon--external-link {% elif params.exit %} icon--exit {% endif %}">{{- params.html | safe if params.html else params.text -}}</span>
+      <span class="btn__inner{% if params.innerClasses %} {{ params.innerClasses }}{% endif %}{% if params.submitType != 'loader' %}{% if params.newWindow %} icon--external-link {% elif params.exit %} icon--exit {% endif %}{% endif %}">{{- params.html | safe if params.html else params.text -}}</span>
```

Comment:
Does this work? Not sure it should be `elif` for `params.exit` because there is no `else` to end the condition.  So it'll require `params.exit` if `params.submitType` isn't defined.  

Do you also need to change these to `if param.foo is defined`?

Reply:
Nah that logic is correct if its not a loader button then it allows you to either add the external or exit classes to the button. The loader if wraps the one with the elifs. Ill add `is defined` though